### PR TITLE
SAK-50415 Syllabus ui responsiveness

### PR DESCRIPTION
**Pull request by Sharadhi98**

### CODE REVIEW

File: syllabus/syllabus-app/src/webapp/syllabus/edit_bulk.jsp [hunk 1]
```diff
@@ -196,7 +196,7 @@
 				<sakai:doc_section>
 					<h:outputText value="#{msgs.newSyllabusBulkForm}"/>
 				</sakai:doc_section>
-				<h:panelGrid columns="1" styleClass="jsfFormTable">
+				<h:panelGrid columns="1" styleClass="jsfFormTable" style="table-layout:fixed; width:100%">
```
Comment (kunaljaykam) [inline syllabus/syllabus-app/src/webapp/syllabus/edit_bulk.jsp:199]:
```suggestion
				<h:panelGrid columns="1" styleClass="jsfFormTable w-100" style="table-layout:fixed;">
```
Let's use bootstrap class where we can

File: syllabus/syllabus-app/src/webapp/syllabus/edit_bulk.jsp [hunk 2]
```diff
@@ -208,15 +208,15 @@
 						</h:panelGroup>
 					</h:panelGroup>
 					<h:panelGroup>
-						<h:selectOneRadio id="radioSingleItem" value="#{SyllabusTool.bulkEntry.addSingleItem}" styleClass="radioSingleItem radioOption radioOptionSelected">
+						<h:selectOneRadio id="radioSingleItem" value="#{SyllabusTool.bulkEntry.addSingleItem}" styleClass="radioSingleItem radioOption radioOptionSelected" style="width:100%">
```
Comment (kunaljaykam) [inline syllabus/syllabus-app/src/webapp/syllabus/edit_bulk.jsp:211]: Update this and others as well

**Review by kunaljaykam — CHANGES_REQUESTED** [review]

File: syllabus/syllabus-app/src/webapp/syllabus/edit_bulk.jsp (hunk 2 above)
Comment (Sharadhi98) [reply syllabus/syllabus-app/src/webapp/syllabus/edit_bulk.jsp]: Thanks! I updated the code and pushed it.

**Comment by adrianfish** [comment]
@Sharadhi98 I'm just rebasing your branch onto master. You have a conflict. I will force push very soon, so don't do anything until I have.

**Review by adrianfish — APPROVED** [review]
Kunal says this is good to go, it's good to go. Thanks Sharadhi.

**Comment by adrianfish** [comment]
@Sharadhi98 Do you know how to fix conflicts in your branches? When your code won't merge cleanly with master? I have no intention to patronise you, not everybody is good with git :)

**Comment by Sharadhi98** [comment]
> @Sharadhi98 Do you know how to fix conflicts in your branches? When your code won't merge cleanly with master? I have no intention to patronise you, not everybody is good with git :)

I'm not sure how to fix that, Adrian. I might have to Google/ChatGPT it and find out!

**Comment by adrianfish** [comment]
It's not too hard and it's a good skill to know.

**Pull request merged**